### ISSUE TITLE
API Crud'r framework creates poor change log message on delete

### ISSUE THREAD

**Issue by mitchell852** [apache/trafficcontrol#5453]
## I'm submitting a ...
-  improvement request (usability, performance, tech debt, etc.)

## Traffic Control components affected ...
-  Traffic Ops

## Current behavior:
Deleting a resource via the API that uses the CRUDr framework produces a poor change log message:

![image](https://user-images.githubusercontent.com/251272/105403440-ff12e980-5be5-11eb-8876-94aee93c9888.png)

## New behavior:
Instead of showing ID three times, include name at a minimum in the change log message and if name doesn't exist on the resource, then i guess show id 3 times. :)

## Minimal reproduction of the problem with instructions:
Delete something via the API that uses the CRUDr framework (i.e. delete a cdn) and observe the poor and redundant change log message created.

**Comment by rawlinp** [comment]
Indeed. In the CRUDer I think we could try to type-assert the `Deleter` into a `Reader` and reuse the object's `Read()` method if so.
